pixeltable/func/callable_function.py

```diff
@@ -48,6 +49,13 @@ def __init__(
         self.__doc__ = self.py_fns[0].__doc__
         super().__init__(signatures, self_path=self_path, is_method=is_method, is_property=is_property)
 
+        if not self.is_polymorphic:
+            # check for _runtime_ctx parameter in our py_fn
+            py_sig = inspect.signature(self.py_fn)
+            self.accepts_runtime_ctx = '_runtime_ctx' in py_sig.parameters
+        else:
+            self.accepts_runtime_ctx = False
```

Comment:
Why do we need this limitation? We could easily check whether all signatures have `_runtime_ctx` as a parameter.

Even better would be to make `accepts_runtime_ctx` a property of the Pixeltable `Signature`, rather than the `CallableFunction`. That's really what it is. Then it would carry over to a particular `FunctionCall` just if it were present in the relevant signature for that `FunctionCall`.